pipeline/release/release.go

```diff
@@ -67,9 +73,13 @@ func description(diff string) string {
 	return result + "\nBuilt with " + string(bts)
 }
 
-func upload(client *github.Client, releaseID int, archive string, ctx *context.Context) error {
-	archive = archive + "." + ctx.Config.Archive.Format
-	file, err := os.Open("dist/" + archive)
+func upload(ctx *context.Context, client *github.Client, releaseID int, archive, format string) error {
+	archive = archive + "." + format
+	var path = filepath.Join("dist", archive)
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return nil
```

Comment:
Makes sense. 
But since the intention here is not clear from just looking at the code it would be nice to add what you just described as a code comment.